html-css/week1/Suman-Order-Desktop/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css">
+        <title>ORDER SUMMARY FORM</title> 
+    </head>
+
+    <body>
+        <!-- Header -->
+        <header class="order">
+            <div class="order_summary">
+                <h1>Order Summary</h1>
+            </div>
+            <div class="order_total">
+                <div class="order_totalpermonth">
+                    <p>Total</p>
+                    <p>$333 / month</p>
+                </div>
+                <hr class="order_line">
+                <div class="order_totalnow">
+                    <p>Total</p>
+                    <p>$100</p>
+                </div>
+            </div>
+        </header>
+
+        <!-- Payment Method -->
+        <section class="payment">
+            <h2 class="payment_method">Payment Method</h2>
+            <div class="payment_option">
+                <button class="payment_button" id="payment_button1" type="button"><img class="payment_img1" src="images/visa.webp" alt="Visa">Credit / Debit</button>
+                <button class="payment_button" id="payment_button2" type="button"><img class="payment_img2" src="images/paypal.png" alt="PayPal">PayPal</button>
+            </div>
+        </section>
+
+        <!-- Card Data Input -->
+        <section class="card">
+            <h2 class="card_info">Card Info</h2>
+            <article class="card_input">
+                <div class="card_data">
+                    <p class="card_text">Card number</p>
+                    <input class="card_number" type="text" placeholder="1234 1234 1234 1234" required>
```

Comment:
Input fields can go hand-in-hand with `<label>` tags, which can be easily associated together, compared to `<p>` tags. For example, by using the `for` and `id` attributes between `<label>` and `<input>` you can easily focus inside the `<input>` when clicking the label. This is especially helpful when dealing with small clickable elements as radio buttons, by making it possible to click on the entire `<label>`
```suggestion
                    <label for="card-number" class="card__label">Card number</label>
                    <input class="card_number" id="card-number" type="text" name="card-number" placeholder="1234 1234 1234 1234" required>
```

Notice, that I've also added a `name` attribute to the `input`. This is used on the JavaScript side to handle the data. More about it in latter sessions ;)